docs/src/appendix/experimental-features.md

```diff
@@ -143,56 +143,156 @@ Consider a Interval with a binary point of 3: aaa.bbb
 | shiftLeftBinaryPoint(2) | a.aabbb |  5 | X | X  | increase the precision |
 | shiftRighBinaryPoint(2) | aaaa.b |  1 | X | X  | reduce the precision |
 
-## Loading Memories in simulation <a name="loading-memories"></a>
+## Loading Memories in simulation or FPGA initialization <a name="loading-memories"></a>
```

Comment:
can you update the table-of-contents above to match this title? Also it's a little awkward as written: Maybe

```
Initializing Memories in Simulation or for FPGA Synthesis
```
?

Reply:
Done! I think "Loading Memories for simulation or FPGA initialization" really sounds better :)
Thanks!